app/routes.py

```diff
@@ -1 +1,245 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import datetime
+import requests
+import json
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+task_list_bp = Blueprint("task_list", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals_list", __name__, url_prefix="/goals")
+#validating the task_id
+def validate_task(task_id):
+    try:
+        task_id = int(task_id)
+    except:
+        abort(make_response({"message": f"Task {task_id} invalid"}, 400))
+    task = Task.query.get(task_id)
+    if not task:
+        abort(make_response({"details": "Invalid Data"}, 404))
+    return task
+
+#validate goal_id
+def validate_goal(goal_id):
+    try:
+        goal_id = int(goal_id)
+    except:
+        abort(make_response({"message": f"Goal {goal_id} invalid"}, 400))
+    goal = Goal.query.get(goal_id)
+    if not goal:
+        abort(make_response({"details": "Invalid Data"}, 404))
+    return goal
```

Comment:
Nice approach to handling an invalid goal 😁 There's a bunch of repeated code here, any ideas of how to refactor?